chart/org.eclipse.birt.chart.engine/schema/aggregatefunctions.exsd

```diff
@@ -11,8 +11,13 @@
    </annotation>
 
    <element name="extension">
+      <annotation>
+         <appInfo>
+            <meta.element />
+         </appInfo>
+      </annotation>
       <complexType>
-         <sequence>
+         <sequence minOccurs="1" maxOccurs="unbounded">
```

Comment:
This change is incorrect. It should be [1,1], and not shown by default.

Reply:
Project "org.eclipes.birt.chart.engine.extension" defines multiple aggregatefunctions in this exact way (i.e. as a sequence of multiple aggregateFunction elements). This contradicts [1,1] and is resolved by setting [1,∞). The concerning file is:
chart\org.eclipse.birt.chart.engine.extension\plugin.xml

Reply:
That looks weird. Even it's defined as [1,1], it's still working properly. But it should be as [1,∞)